calcom/cal.com
# Fixes #2178

## What does this PR do?
<img width="433" alt="image" src="https://user-images.githubusercontent.com/19562383/159372558-4216e485-8b51-4b0c-8124-1a2c883a35b0.png">

I changed the RGBA values in primary50. So now, when you select an option, the background color is _bg-gray-300_ and its text color is white. 
<img width="402" alt="image" src="https://user-images.githubusercontent.com/19562383/159373895-05bea335-62e8-42f9-acc7-2254b56aebaf.png">

I took background color references from https://tailwindcss.com/docs/background-color.


Fixes # 2178

## Type of change




## How should this be tested?



Run the app and when you are in step 3 of getting-started page, select different times and you are going to see perfectly the time.
